client/gutenberg/extensions/vr/editor.js

```diff
@@ -0,0 +1,107 @@
+/** @format */
```

Comment:
Should we add this block to the `jetpack` preset so we can load it?

Reply:
I didn't add it deliberately:

> I'm not yet adding this to the `jetpack` preset, since it's not in good enough shape yet [...]

While I wanted to remove it from the JP repo, I didn't want to commit us to polishing it for inclusion with Jetpack, since other blocks have higher priority.